### PR TITLE
CSS grid was implemented

### PR DESCRIPTION
**Pull request by pabloqv10**
## Summary

- CSS grid was implemented

## Trello Card

[https://trello.com/c/ZOY0OtIb/12-redise%C3%B1o-de-la-p%C3%A1gina](url)

## Screenshots
css-grid

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -3,9 +3,13 @@ $break-medium: 1024px;
...
   background-color: $white;
   font-family: 'Roboto', sans-serif;
-  max-width: 900px;
-  padding: 24px;
+  max-width: 853px;
+  height: 570px;
+  padding: 32px 32px 56px 32px;
   width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```
Comment (manueltuero) [inline scss/detail.scss]: Use `repeat(5, 1fr)`;

File: scss/detail.scss [hunk 2]
```diff
@@ -3,9 +3,13 @@ $break-medium: 1024px;
...
+  grid-template-rows: 1fr 1fr 1fr;
```
Comment (manueltuero) [inline scss/detail.scss]: same

File: scss/detail.scss [hunk 3]
```diff
@@ -115,17 +128,29 @@ $break-medium: 1024px;
   }
 
   .book-image {
-    align-self: center;
-    margin: 0 0 21px 0;
+    text-align: center;
+    margin: 0 0 6px 0;
```
Comment (manueltuero) [inline scss/detail.scss:132]: margin-bottom

**Review by manueltuero — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 1 above)
Comment (pabloqv10) [reply scss/detail.scss]: Done

File: scss/detail.scss (hunk 2 above)
Comment (pabloqv10) [reply scss/detail.scss]: Done

File: scss/detail.scss (hunk 3 above)
Comment (pabloqv10) [reply scss/detail.scss:132]: @manueltuero I use this property for reseat margin when the screen use a media query

**Review by pabloqv10 — COMMENTED** [review]
Code review fixes

File: scss/detail.scss [hunk 4]
```diff
@@ -3,9 +3,13 @@ $break-medium: 1024px;
...
   font-family: 'Roboto', sans-serif;
-  max-width: 900px;
-  padding: 24px;
+  max-width: 853px;
+  height: 570px;
+  padding: 32px 32px 56px 32px;
   width: 100%;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: repeat(3, 1fr);
```
Comment (Dorian30) [inline scss/detail.scss:12]: Don't need to change this. In future projects you can also use grid areas. I think it's a more semantic approach to do this and easier to follow. Good work!! :rocket:

**Review by Dorian30 — APPROVED** [review]

**Review by manueltuero — APPROVED** [review]

**Pull request merged**